src/main/java/seedu/lifeasier/storage/FileStorage.java

```diff
@@ -94,15 +94,21 @@ private void checkForNotesSaveFile(File saveFileNotes) {
         //Create new save file if task save file does not exist
         if (!saveFileNotes.exists()) {
             createNewSaveFile(saveFileNotes);
+            return;
         }
+        //Read note save file
+        noteStorage.readNotesSave(filePathNotes);
     }
 
     private void checkForTaskSaveFile(File saveFileTasks) {
```

Comment:
Could be renamed to make it consistent with checkForNotesSaveFile and other variables
```suggestion
    private void checkForTasksSaveFile(File saveFileTasks) {
```